exercises/pangram/example.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PANGRAM_H
+#define PANGRAM_H
+#include <string>
+
+namespace pangram
+{
+    
+   bool is_pangram(std::string s);
```

Comment:
The repo convention is to not indent namespace contents. So this whole namespace should be:

```
namespace pangram
{

bool is_pangram(std::string s);

}
```